src/net/RTP/RTPSession.cs

```diff
@@ -261,12 +243,6 @@ public class RTPSession : IMediaSession, IDisposable
         /// </summary>
         public bool UseSdpCryptoNegotiation { get; private set; } = false;
 
-        /// <summary>
-        /// If this session is using a secure context this flag MUST be set to indicate
-        /// the security delegate (SrtpProtect, SrtpUnprotect etc) have been set.
-        /// </summary>
-        public bool IsSecureContextReady { get; private set; } = false;
-

```

Comment:
Can this property stay? As it's public there may be people relying on it.

Reply:
I can attempt to keep it, but it's meaning is a bit different and harder to figure out now since SrtpProtect/Unprotect etc. are now split between audio and video. This means I would have to figure out what media types we are using in the session before being able to accurately say if the Secure Context is ready.

My main reason for removing it was to change it for a function, in order to keep using it in the areas it was inside of RTPSession. I can try and bring it back, and set it at a different time

Reply:
It's reasonable to change the behaviour. That can go in the release notes. Couldn't it only return true once both the audio and video secure contexts are ready? That's likely to reflect the closest way people are using it at the moment.

Reply:
I did that initially, but it caused issues wherever the RTPSession would use it, if the session was audio only. 

What could work is to change the internal usages to something else (that way we don't impact the public member) and we can keep this as it was before (either, any of audio or video are ready, or only both for example)

Reply:
If tthe session is audio only shouldn't the `IsSecureContextReady` just get set to `true` as soon as the single SRTP context is ready?

> What could work is to change the internal usages to something else (that way we don't impact the public member) and we can keep this as it was before (either, any of audio or video are ready, or only both for example)

That sounds fine.

Reply:
I'll go ahead with that solution. I'll keep the IsSecureContextReady  property as close as possible to what it was before

Reply:
I've done a pass, let me know what you think of the solution

Reply:
Looks good.